doc/cephadm/version-tracker.rst

```diff
@@ -0,0 +1,134 @@
+.. _cli-version-tracker:
+
+=========================
+Version Tracking for Ceph
+=========================
+
+Cephadm tracks all cluster version history and allows accessing 
```

Comment:
This wording is a bit vague.  I suggest:

Cephadm tracks the history of cluster daemon versions and allows retrieval of this information. There are two CLI commands to manage this information.  Daemon version information from before the introduction of this feature is not available, which means that tracking will begin with the Umbrella release, or possibly Tentacle update releases.